src/test/java/com/mongodb/jdbc/MongoResultSetTest.java

```diff
@@ -1558,19 +1545,26 @@ void testEmptyResultSetWhenGetMetadataCalledFirst() throws SQLException {
     }
 
     @Test
-    void differentMetaDataOnDifferentRows() throws SQLException {
+    void sameMetaDataOnDifferentRowsEvenWithDifferentBsonTypes() throws SQLException {
```

Comment:
I've changed this test, because the old test was testing something that could never happen: getting different metadata info for the same column across multiple rows

Reply:
Yep, it used to be theoretically possible, now it is not.